packages/theme/src/themes/fram-theme/theme.ts

```diff
@@ -16,6 +16,20 @@ const contrastColor = (
   };
 };
 
+// Currently exactly the same as in the AtB theme.
+// The colors can be changed, but should follow standard practice:
+// no entry - dark/black
+// no parking - red
+// slow zone - yellow
+// allowed - blue
```

Comment:
I think this is useful documentation globally. Maybe put it next to the `GeofencingZoneCodes` type?